sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosAsyncContainer.java

```diff
@@ -661,6 +663,68 @@ public CosmosAsyncConflict getConflict(String id) {
                    }).map(offerResourceResponse -> offerResourceResponse.getResource().getThroughput());
     }
 
+    /**
+     * Replace the throughput .
+     *
+     * @param throughputProperties the throughput properties
+     * @return the mono containing throughput response
+     */
+    public Mono<ThroughputResponse> replaceThroughput(ThroughputProperties throughputProperties) {
+        return this.read()
+                   .flatMap(response -> this.database.getDocClientWrapper()
```

Comment:
Is the indentation right?

It looks very nested and deep. @moderakh ?

Reply:
It looks a bit weird, but this is correct. Also this is the best intellij could do :)

Reply:
The indentation is correct. The nesting is the result of reactive-stream chaining and lambdas